resources/terminology.md

```diff
@@ -0,0 +1,75 @@
+| **Reversal**              | Used to reverse a payment. It is only possible to reverse a payment that has been captured and not yet reversed.                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| **Sale**    | A [one-phase][one-phase-url] transaction where the amount is settled when the transaction has succeded. Payment instruments using sales transactions are Swish and Direct Bank Debit.                                                                                                                                                                                                                                                                                                                                                                                               |
+| **Two-phase payment flow**            | A payment done in [two steps] [two-phase-payments]. Authorization and capture. The most common payment instrument using two-phase payments is card payments.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| **Verify**     | The payment operation that initiates a [verification][verify-url] payment process. It is a payment that lets you post verifications to confirm the validity of card information, without reserving or charging any amount. This option is used to generate a payment- or recurrence token, that can be used in a recurring payments scenarios or for one-clickpayments, without charging the card in the process. |
+
+[admin-manual]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin/
+[admin-portal]: https://admin.externalintegration.payex.com/psp/login
+[admin-getting-started]: https://developer.payex.com/xwiki/wiki/developer/view/Main/ecommerce/resources/admin/
```

Comment:
We don't need to link to info about the "old" admin in the old developer portal pages.